components/atoms/NextSection/NextSection.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from "react";
+import classnames from "classnames";
+import Link from "next/link";
+import Image from "next/image";
+
+import styles from "./NextSection.module.scss";
+
+type PageProperty = {
+  title: string;
+  path: string;
+  className?: string;
+};
+
+const NextSection: FC<PageProperty> = ({ title, path, className }) => {
+  return (
+    <Link href={path} passHref>
+      <div className={classnames(styles.section, className)}>
+        <div className={classnames(styles.next, className)}>
+          Next
+          <Image
+            alt="read more"
+            src="/images/common/icons/readmore.svg"
+            width="24"
+            height="24"
+          />
+        </div>
+        <div className={classnames(styles.title, className)}>{title}</div>
+      </div>
+    </Link>
+  );
+};
```

Comment:
Suggest that injected `className` just to the outermost layer component.
If someday we want to change some looks for `NextSection`, we can create a `type` property to let the logic of style be in the component. That can avoid making coupling in called components.


```suggestion
const NextSection: FC<PageProperty> = ({ title, path, className }) => {
  return (
    <Link href={path} passHref>
      <div className={classnames(styles.section, className)}>
        <div className={styles.next}>
          Next
          <Image
            alt="read more"
            src="/images/common/icons/readmore.svg"
            width="24"
            height="24"
          />
        </div>
        <div className={styles.title}>{title}</div>
      </div>
    </Link>
  );
};
```